articles/understanding-the-basics-of-responsive-web-design/index.md

```diff
@@ -0,0 +1,118 @@
+  
+### How to Get Started With RWD
+If you already have an active website, you should test it to establish whether it is mobile-friendly or responsive to various devices. Visit [this page](https://search.google.com/test/mobile-friendly) to perform this test. When you get to the page, enter the URL for your website and click on ‘test URL’. The tool will then determine whether your site is responsive or not. If the website is not responsive, follow this brief guide to have an overview of how to create a responsive one. 
+
+The following is a brief overview of the steps that should be taken to build a responsive WordPress website from scratch. 
+
+**Step 1:** Choose a domain name
+You should choose a good domain name that can enable customers to locate your responsive website. The name should represent your brand or business. 
+
+**Step 2:** Choose a hosting plan
+There is a wide variety of hosting services for WordPress websites. Choose a hosting plan that corresponds to your financial capabilities and business needs. Click [here](https://websitesetup.org/best-wordpress-hosting-performance/#:~:text=10%20Best%20WordPress%20Hosting%20Services%201%20Bluehost%20%28www.Bluehost.com%29,Site5%20%28www.Site5.com%29%209%20DreamHost%20%28www.DreamHost.com%29%2010%20iPage%20%28www.iPage.com%29) to get a list of companies that can host your WordPress website.  
+
+**Step 3:** Images and content
+Prepare all the images and content that will be used on your website. Doing this in advance enables you to replace the demo content in your purchased theme with your ready content. 
+
+**Step 4:** Select a responsive WordPress Theme
+You can purchase a responsive WordPress theme [here](https://themeforest.net/popular_item/by_category?category=wordpress&_ga=2.256398347.1451571840.1610998232-961460759.1609948334). The theme should be selected based on the features, customization options, and the design. An appealing design should be chosen to attract customers to the responsive website. 
+
+**Step 5:** Install WordPress
+When you select a hosting plan, the hosting company will send a link to the email you used for registration. This link will take you to the hosting dashboard. When you fill in your credentials, you will find a section called ‘WordPress install’. Click on the WordPress icon and go through the instructions. 
+
+Another screen will pop up requiring you to fill in details such as your site’s description and desired credentials. After filling in all this information, click on ‘install’. Installing WordPress means that you can now access the WordPress dashboard using the registered credentials. 
+
+**Step 6:** Installing a theme
+In this step, you need to install the WordPress theme that you chose in step 4. In the WordPress dashboard, follow this selection sequence: **appearance**-> **themes**-> **add new**. Click on 'upload theme'. Click on 'install' to begin the installation. When this is done, click on 'activate'. 

```

Comment:
You could attach screenshots here to support your instructions

Reply:
Done.